emails/tests/views_tests.py

```diff
@@ -217,21 +217,21 @@ def test_sns_message_with_hard_bounce(self):
 
         _sns_notification(BOUNCE_SNS_BODIES["hard"])
 
-        profile = self.user.profile_set.get()
-        assert profile.last_hard_bounce >= pre_request_datetime
+        self.user.refresh_from_db()
```

Comment:
question (non-blocking): does `refresh_from_db()` also refresh related field data? Looks like most of these tests are written such that they might pass even if the data was stale?